src/api/MetricsApi.ts

```diff
@@ -33,7 +33,6 @@ interface IMetricEvent {
 
 export default class MetricsApi implements IMetricsApi {
   private metricsList = spWebContext.lists.getByTitle("Metrics");
```

Comment:
Should the private keyword be added to where the userApi is used in the Dev class?